src/pages/identity/activity-boxes/Apps.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GenericItem from './shared/GenericItem';
+import GenericList from './shared/GenericList';
+import { Svg } from '@nomios/Web-uikit';
+import styles from './Apps.css';
+import appIcon from '../../../shared/media/backgrounds/background-pattern-1440p.png';
+
+const globeSvg = import('../../../shared/media/illustrations/globe.svg');
```

Comment:
Give the same webpack chunk name as we are doing to other illustrations: `svg-illustrations-sprite`